src/xGadgetron/cGadgetron/include/sirf/Gadgetron/gadgetron_data_containers.h

```diff
@@ -119,7 +127,18 @@ namespace sirf {
         mutable bool have_header_;
 	};
 
-    class KSpaceSorting
+    /*!
+    \ingroup Gadgetron Data Containers
```

Comment:
probably no space

Reply:
ok so this will be addressed by #886 I assume